src/pages/teachers/curriculum/index.tsx

```diff
@@ -24,24 +21,48 @@ import Breadcrumbs from "@/components/SharedComponents/Breadcrumbs/Breadcrumbs";
 import Illustration from "@/components/SharedComponents/Illustration/Illustration";
 import Cover from "@/components/SharedComponents/Cover/Cover";
 import { getSizes } from "@/components/SharedComponents/CMSImage/getSizes";
-import OwaLink from "@/components/SharedComponents/OwaLink/OwaLink";
-import CMSClient from "@/node-lib/cms";
-import BrushBorders from "@/components/SharedComponents/SpriteSheet/BrushSvgs/BrushBorders/BrushBorders";
-import { blogToPostListItem } from "@/components/GenericPagesViews/BlogIndex.view";
-import { serializeDate } from "@/utils/serializeDate";
-import PostListItem from "@/components/SharedComponents/PostListItem";
-import { SerializedBlogPostPreview } from "@/common-lib/cms-types";
 import { isExamboardSlug } from "@/pages-helpers/pupil/options-pages/options-pages-helpers";
 import HomepageCurriculumLandingHero from "@/components/GenericPagesComponents/HomepageCurriculumLandingHero";
+import CurricInfoCard from "@/components/CurriculumComponents/CurricInfoCard";
+import CurricQuote from "@/components/CurriculumComponents/CurricQuote";
 
 export type CurriculumHomePageProps = {
   curriculumPhaseOptions: SubjectPhasePickerData;
-  posts: SerializedBlogPostPreview[];
 };
 
+const StyledContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
```

Comment:
I think we can move most of this into a `<OakBox/>` so

```jsx
const SpecialBreakpointLogic = styled.div`
  flex-direction: column;
  
  @media (min-width: 1000px) {
    flex-direction: row;
  }
`

<SpecialBreakpointLogic>
  <OakBox $pa={[...]} $gap={[...]} ...>
     {/* inner bit... */}
  </OakBox>
</SpecialBreakpointLogic>
``` 

This means we're using the tokens for the most possible and only using special logic where we need to